changes/5568.changed

```diff
@@ -0,0 +1 @@
+Added url link to the total device count number under device family.
```

Comment:
Wrong change fragment? :-) Should be a `6867.documentation` describing the below fix.